src/Microsoft.Data.Sqlite/project.json

```diff
@@ -9,10 +16,7 @@
     ],
     "xmlDoc": true,
     "embed": {
-      "include": [
-        "Properties/Microsoft.Data.Sqlite.rd.xml",
-        "Strings.resx"
```

Comment:
Why does it need to be embedded? The compiler embeds the transformed `*.resources` file.